packages/mdc-textfield/bottom-line/_mixins.scss

```diff
@@ -17,19 +17,19 @@
 //Public
 
 @mixin mdc-text-field-bottom-line-color($color) {
-  &:not(.mdc-text-field--disabled) {
+  &:not(.mdc-text-field--disabled):not(.mdc-text-field--outlined):not(.mdc-text-field--textarea) {
```

Comment:
Don't we say in our README to never use a `mdc-text-field__bottom-line` element within a `mdc-text-field--outlined` element? or `mdc-text-field--textarea`?

Reply:
Maybe. Since they share the mdc-text-field__input, the bottom line color is automatically set, resulting in some weird behavior on outline and textfield if we don't stop it here. This mixin applies border-bottom-color, and outline applies border-color, resulting in a weird line

Without
![screenshot from 2018-01-19 17-25-43](https://user-images.githubusercontent.com/34519388/35178449-f8eea1fc-fd3d-11e7-9e03-25616dc4c7e2.png)

With
![screenshot from 2018-01-19 17-26-45](https://user-images.githubusercontent.com/34519388/35178451-010e90ae-fd3e-11e7-9e00-aab2e608a59b.png)